app/src/main/java/com/jetbrains/handson/mpp/mobile/MainActivity.kt

```diff
@@ -14,7 +14,16 @@ class MainActivity : AppCompatActivity(), ApplicationContract.View {
         presenter.onViewTaken(this)
     }
 
-    override fun setLabel(text: String) {
-        findViewById<TextView>(R.id.main_text).text = text
+    override fun setStationNames(stationNames: List<String>) {
+        //To DO
```

Comment:
Tiny nit but I'd prefer todos written like:
`//TODO: Thing that needs to be done`